render.py

```diff
@@ -653,6 +609,36 @@ def invoke(self, context, event):
         return {'RUNNING_MODAL'}
 
 
+class ShowImage(Operator):
+    """Show render image"""
+
+    bl_idname = f"{HANA3D_NAME}.show_image"
+    bl_label = ""
+
+    index: IntProperty(
+        name='index',
+    )
+
+    @execute_wrapper
+    def execute(self, context):
+        asset_props = utils.get_upload_props()
+        filepath = asset_props.render_list[self.index]['file_path']
+
+        image = bpy.data.images.load(filepath, check_existing=True)
+        image.name = asset_props.render_list[self.index]['name']
+        asset_props.render_list[self.index]['name'] = image.name
+
+        bpy.ops.render.view_show('INVOKE_DEFAULT')
+        try_again = True
+        while try_again:

```

Comment:
Esse `while` é realmente necessário? `image` já deveria estar carregado. Teria alguma situação que bloqueia de escrever no `bpy.context.area.spaces.active.image`?

Reply:
então, também não sei porque muitas vezes ele ainda não abriu a janela que o `bpy.ops.render.view_show` chama. aí ele dá erro quando eu tento escrever em `bpy.context.area.spaces.active.image`

Reply:
hmm pode estar relacionado com você estar usando `bpy.context.area.spaces.active`, com o Blender tentando colocar a imagem no `space` ativo da `area` do atual `context`, mas se o `context` que ele encontrar quando rodar esse código já não for do `render.view_show` ele não vai conseguir por a imagem, ou pior ainda se o `context` que ele encontrar também tiver a propriedade `image` vai colocar uma imagem nele.

O que eu acho que talvez resolva é achar o `space` do `render.view_show` e colocar a imagem nele direto, dando uma testada aqui parece que ele estaria em `bpy.data.screens['temp'].areas[0].spaces[0]`.